ads/google/adsense.js

```diff
@@ -21,7 +21,7 @@ import {checkData} from '../../src/3p';
  * @param {!Object} data
  */
 export function adsense(global, data) {
-  checkData(data, ['adClient', 'adSlot', 'adHost']);
+  checkData(data, ['adClient', 'adSlot', 'adHost', 'adtest', 'tagOrigin']);
```

Comment:
Yes, that is awkward, but it reflects that the amp-ad option is spelled "data-adtest", rather than "data-ad-test". That is inconsistent with other AdSense ad parameters, "data-ad-client", etc. But it is the way it is spelled as a current AdSense parameter. So it is consistent with inconsistent current practice.

You can see it spelled the same way in AMP for aduptech,js.